.ibm/pipelines/utils.sh

```diff
@@ -410,8 +396,11 @@ apply_yaml_files() {
     done
 
     DH_TARGET_URL=$(echo -n "test-backstage-customization-provider-${project}.${K8S_CLUSTER_ROUTER_BASE}" | base64 -w 0)
-
-    for key in GITHUB_APP_APP_ID GITHUB_APP_CLIENT_ID GITHUB_APP_PRIVATE_KEY GITHUB_APP_CLIENT_SECRET GITHUB_APP_JANUS_TEST_APP_ID GITHUB_APP_JANUS_TEST_CLIENT_ID GITHUB_APP_JANUS_TEST_CLIENT_SECRET GITHUB_APP_JANUS_TEST_PRIVATE_KEY GITHUB_APP_WEBHOOK_URL GITHUB_APP_WEBHOOK_SECRET KEYCLOAK_CLIENT_SECRET ACR_SECRET GOOGLE_CLIENT_ID GOOGLE_CLIENT_SECRET K8S_CLUSTER_TOKEN_ENCODED OCM_CLUSTER_URL GITLAB_TOKEN DH_TARGET_URL; do
+    local RHDH_BASE_URL=$(echo -n "https://${release_name}-backstage-${project}.${K8S_CLUSTER_ROUTER_BASE}" | base64 | tr -d '\n')
+    if [[ "$JOB_NAME" == *aks* || "$JOB_NAME" == *gke*  ]]; then
```

Comment:
Hi @subhashkhileri I would like to make this method generic, without relying on a direct connection to the job names. That's why the files are currently separated by jobs. Would it be possible to remove this `if` and place this part directly within `gke` and `aks`?

Reply:
@gustavolira thing is even gke and aks call same `apply_yaml_files` so that's the reason it has if condition. another approach could be to accept `RHDH_BASE_URL` as a parameter in apply_yaml_files.

Reply:
instead RHDH_BASE_URL I am getting release_name in parameter so may be it can be used somewhere else in future but right now its only used to construct RHDH_BASE_URL